pkgs/development/python-modules/pcre2-py/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  build,
+  bzip2,
+  cmake,
+  cython,
+  editline,
+  gitpython,
+  pytestCheckHook,
+  python3Packages,
+  buildPythonPackage,
+  fetchFromGitHub,
+  haskellPackages,
+  lib,
+  libedit,
+  libz,
+  pcre2,
+  scikit-build,
+  setuptools,
+  twine,
+  readline,
+  requests,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "pcre2-py";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "grtetrault";
+    repo = "pcre2.py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-mivw+IITO6GTj3OnoE+nb4K2nZQRWk8OWWXIGnIYn/o=";
+    fetchSubmodules = true;
+  };
+
+  dontUseCmakeConfigure = true;
+
+  build-system = [
+    cython
+    scikit-build
+    setuptools
+    wheel
+  ];
+
+  dependencies =
+    [
+      haskellPackages.bz2
+      haskellPackages.memfd
+    ]
+    ++ [
+      build
+      bzip2
+      # to avoid error in required build-dependencies cmake
```

Comment:
Why not patch it out if it just fixes an error?

Reply:
whats best approach for patching it out.